<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="paused" default="true"/>
  <arg name="gui" default="true"/>
  <arg name="debug" default="false"/>
  <arg name="world" default="$(find my_simulations)/world/corridor.world"/> 

  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(arg world)"/>
    <arg name="paused" value="$(arg paused)"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="$(arg gui)"/>
    <arg name="debug" value="$(arg debug)"/>
  </include>

  <include file="$(find my_simulations)/launch/wheelchair_base.launch" >
    <arg name="X" value="-22"/>
    <arg name="Y" value="2"/>
    <arg name="yaw" value="0"/>
  </include> 

  <!-- <include file="$(find my_simulations)/launch/wheelchair_base.launch" >
    <arg name="X" value="1"/>
    <arg name="Y" value="2"/>
    <arg name="yaw" value="3"/>
  </include>  -->

  <!--include file="$(find my_simulations)/launch/move_baseSafe.launch">
    <arg name="namespace" value=""/>
  </include--> 

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find my_simulations)/includes/rviz_config/auto_rviz.rviz" />
  
  <!--group ns="/robot_1">
    <include file="$(find my_simulations)/launch/robot.launch.xml">
      <arg name="robot_name" value="robot_1"/>
      <arg name="init_pose" value="-x 0.0 -y 0.0 -z 0.0"/>
    </include>
  </group-->

  <!--node respawn="false" pkg="my_simulations" type="key_teleop.py" name="key_teleop" /-->
</launch>
